infrastructure/dynamodbkutter.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
+  name     = "kutter-table"
+  billing_mode   = "PROVISIONED"
+  read_capacity  = 5
+  write_capacity = 5
+  hash_key = "objectId"
```

Comment:
why not id?

Reply:
I want to write all the keys in the database like that, instead of underscore I will write the first letter of the next word with uppercase. Here my metadata for the photo objects: https://github.com/Chufle/kutter/issues/5#issuecomment-1324947453
I see this kind of writing in a lot of json files and APIs. Not a good idea?

Reply:
I leave it like this